src/ResetPasswordHelperInterface.php

```diff
@@ -6,13 +6,38 @@
 
 /**
  * @author Jesse Rushlow <jr@rushlow.dev>
- * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Ryan Weaver <ryan@symfonycasts.com>
  */
 interface ResetPasswordHelperInterface
 {
+    /**
+     * Generate & persist a new password reset token that can be provided to the user
+     *
+     * ResetPasswordHelper persists a ResetPasswordRequest object which contains
+     * the hashed token. The ResetPasswordToken object returned by this method
+     * contains the user token which is used to select and verify a persisted
+     * request object.
```

Comment:
I think we can be a bit more generic - i.e. talk less about the implementation inside ResetPasswordHelper and more about the contract of what this method needs to do - e.g.

> Generate a new ResetPasswordToken that can be provided to the user.
>
> This method must also persist the token information to storage so that
> the validateTokenAndFetchUser() method can verify the token validity
> and removeResetRequest() can eventually invalidate it by removing it
> from storage.

Reply:
done